### PR TITLE
Release notes 12-9

### PR DESCRIPTION
**Pull request by srpearce**
@mhoeber @lamagnifica @catong Please review.

@clintonb Since you're release lord (should that be capitalized?) this week, you get a sneak peek at the release notes. You don't have to do a thorough review, but feel free to read through them and let me know if anything seems amiss.

### CODE REVIEW

File: en_us/release_notes/source/12-09-2014.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+******************************
+edX Learning Management System
+******************************
+
+* On the Instructor Dashboard, some tooltips appeared far away from the
+  associated controls. This problem has been fixed. (TNL-558)
+
+* In the course discussions, the **Vote** button was still available even when
+  the thread was closed. This problem has been fixed. (TNL-152)
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst:46]: Please add per DOC-1307 and DOC-1325:

---

edX Insights

---
- The enrollment activity chart, metrics, and report are now available for courses created in Studio before December 4, 2013. Previously, those courses were affected by a change in the method that edX used to track student enrollments. To make enrollment data available in Insights for the affected courses, enrollment history was reconstructed from database tables and user activity logs. EdX Insights now reports enrollment data for the affected courses beginning with November 11, 2013.
- The enrollment activity chart is now presented as a stacked area chart. The filled area
  represents the total number of enrolled learners on a particular date. For
  courses that offer more than one enrollment option or track, different colors
  represent the number of students who are enrolled with each option.

File: en_us/release_notes/source/12-09-2014.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+each release of the edX Platform on `edx.org`_ and `edX Edge`_.
+
+You can also sign up for the `edX Release Announcements mailing list`_ to get
+an email message when we update the edX Course Staff Release Notes.
+
+*************
+edX Studio
+*************
+
+* On the **Grading** page for a course, the button to add grade sections to a
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst]: I don't think this fix needs to be included. (Unless the ticket was opened based on external feedback.)

File: en_us/release_notes/source/12-09-2014.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  course was missing its plus sign icon. This problem has been fixed. (TNL-890)
+
+* When users exported a course, an error message sometimes appeared. This
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst]: Was the message spurious? It's hard to tell right now what got fixed.

File: en_us/release_notes/source/12-09-2014.rst (hunk 2 above)
Comment (catong) [inline en_us/release_notes/source/12-09-2014.rst]: I agree; probably calls more attention to a bug that few would have noticed. :-)

File: en_us/release_notes/source/12-09-2014.rst [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+Students who enroll in a verified course must currently complete the ID
+verification process before completing the payment process. Soon, the ID
+verification and payment processes will be separate.
+
+**************************
+edX Documentation
+**************************
+
+* A link to the new `edX101`_ course was added to the preface that appears at
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst]: Could we give this a little more of a marketing twist? Maybe add something like, "This high-level overview of the course creation and delivery process introduces the basic course components and problem types available to course authors. Enroll in it now on edx.org {link to course}."

**Comment by lamagnifica** [comment]
My initial contributions complete. Will review again.

File: en_us/release_notes/source/12-09-2014.rst (hunk 1 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst:46]: Sure thing. Can you remind me where this info comes from? When I do release notes I check the release page and the PRs in the documentation repo, but I guess Insights has its own repo that I should also be checking? Or is there a quick way to check completed sprints (I think there's a way to do it but I don't remember it off the top of my head)?

File: en_us/release_notes/source/12-09-2014.rst (hunk 2 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst]: OK.

File: en_us/release_notes/source/12-09-2014.rst (hunk 3 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst]: I'll clarify.

File: en_us/release_notes/source/12-09-2014.rst (hunk 1 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst:46]: Actually, on second thought, these seem a bit long and detailed for release notes. Do you mind if I cut them down a bit?

Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst:46]: I set up additional subtasks and add them to the release notes task so that they don't get forgotten. I don't know why, probably because they seem to be Insights-related rather than release notes-related, but they get assigned to me rather than to whoever is writing the release notes. And therefore they get missed.
Sure, edit at will, that's why I requested a 2nd review opportunity.

File: en_us/release_notes/source/12-09-2014.rst (hunk 4 above)
Comment (mhoeber) [inline en_us/release_notes/source/12-09-2014.rst]: agree.  I think it can even warrant it's own section, not just under doc updates.

File: en_us/release_notes/source/12-09-2014.rst (hunk 1 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst:46]: See below (including questions)...

File: en_us/release_notes/source/12-09-2014.rst (hunk 4 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst]: OK. Will make this change tomorrow. I'm officially stopping work for the night and I forgot to do this.

File: en_us/release_notes/source/12-09-2014.rst [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+  for courses created in Studio before December 4, 2013. Previously, those
+  courses were affected by a change in the method that edX used to track student
+  enrollments. To make enrollment data available in Insights for the affected
+  courses, enrollment history was reconstructed from database tables and user
+  activity logs. EdX Insights now reports enrollment data for the affected
+  courses beginning with November 11, 2013.
+
+  REVISION: EdX Insights now reports enrollment data, including the enrollment
+  activity chart, metrics, and report, for courses created on or after November
+  11, 2013.
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst]: no. The affected courses were created on any date prior to December 4. The data for those courses is now available. The data that is available, regardless of when the course was created if before Dec 4th, begins on November 11.
better: 1st and last sentences, cut out the middle:
The enrollment activity chart, metrics, and report are now available
for courses created in Studio before December 4, 2013. EdX Insights 
now reports enrollment data for the affected courses beginning with 
November 11, 2013.

File: en_us/release_notes/source/12-09-2014.rst [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+  .. Is all the material in the original bullet point covered in our
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst]: no. 
The strategy on my end was to describe what necessitated this change once, when the change was made, and put it in the release notes. In the doc, I describe what _is_ (essentially, the 1st and last sentences as above).

File: en_us/release_notes/source/12-09-2014.rst [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+  track, different colors represent the number of students who are enrolled with
+  each option.
+
+  REVISION: The enrollment activity chart is now a stacked area chart. The chart
+  indicates the total number of enrolled learners for each possible track on a
+  particular date.
+
+  .. I'm not sure all our readers will know what a stacked area chart is (I 
+    had to google it). Is the additional information in the original bullet
+    covered in our docs, and, if not, can we add it - including a screen shot?
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst]: yes.
unfortunately, I am stuck in the midst of a git squash attempt gone wrong, and I can't switch to the branch that will have that change in it so that I can provide the reference to you. If I get a chance between meetings later today, and my git issue gets resolved, I'll try again. If not, see the edx-analytics-dashboard repo, ahodges/DOC1039 branch.
Perhaps:
For courses that offer more than one enrollment option or track, the enrollment activity chart now uses different colors to represent the number of students who are enrolled with each option.

File: en_us/release_notes/source/12-09-2014.rst (hunk 5 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst]: OK. I think that was my original plan, but my brain shut down (was feeling really sick last night). Thanks for the revision.

File: en_us/release_notes/source/12-09-2014.rst (hunk 6 above)
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst]: Yeah, I can understand wanting people to know what led to the change. I don't think we explain the methods behind our madness in most cases unless people explicitly ask, though, and this seems like more information than we usually include in release notes.

File: en_us/release_notes/source/12-09-2014.rst [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+These edX Course Staff Release Notes are always available and are updated with
+each release of the edX Platform on `edx.org`_ and `edX Edge`_.
+
+You can also sign up for the `edX Release Announcements mailing list`_ to get
+an email message when we update the edX Course Staff Release Notes.
+
+******************************
+edX Resources
+******************************
```
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst:21]: If someone has a better idea for a heading here, let me know. I wanted to highlight the new edX101 but wasn't sure what to call it.

**Comment by srpearce** [comment]
@mhoeber @lamagnifica @catong Please review this draft. I moved the edX101 announcement to the top of the release notes.

File: en_us/release_notes/source/12-09-2014.rst [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+
+* A new version of the `edX101`_ course has been released on edx.org. This high-level
+  overview of the course creation and delivery process includes guidance for
+  adapting traditional courses to the edX platform and introduces the basic
+  course components and problem types that course authors can use.
```
Comment (srpearce) [inline en_us/release_notes/source/12-09-2014.rst:26]: Thinking about adding "A link to this course appears in the preface at the beginning of each edX documentation resource" or similar - thoughts?

File: en_us/release_notes/source/12-09-2014.rst [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+
+* The enrollment activity chart, metrics, and report are now available for
+  courses created in Studio before December 4, 2013. EdX Insights now reports
+  enrollment data for the affected courses beginning with November 11, 2013.
+
+* The enrollment activity chart is now a stacked area chart that indicates the
+  total number of enrolled learners on a particular date. For courses that offer
+  more than one enrollment option or track, the enrollment activity chart uses
+  different colors to represent the number of students who are enrolled with
+  each option.
```
Comment (mhoeber) [inline en_us/release_notes/source/12-09-2014.rst:59]: shouldn't these two bullets be under Insights heading?

File: en_us/release_notes/source/12-09-2014.rst (hunk 9 above)
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst:26]: I don't think you need that, but an invitation to enroll, or info that it only takes an hour or two to take would be nice.

File: en_us/release_notes/source/12-09-2014.rst [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+
+**************************
+Coming Soon
+**************************
+
+Students who enroll in a verified course must currently complete the ID
+verification process before completing the payment process. Soon, the ID
+verification and payment processes will be separate.
```
Comment (lamagnifica) [inline en_us/release_notes/source/12-09-2014.rst:67]: separate confuses me each time I read this. Is it currently a single process? If so, maybe rephrase in the first sentence so that the structure doesn't emphasize two sequential steps. Or, maybe it was always two processes and now they can be completed independently of each other?
